### PR TITLE
[sqlite3] Sqlite3 geopoly and json1 features

### PR DESCRIPTION
**Pull request by sanjaychugh**
**Describe the pull request**
This change only modifies the sqlite3 port by adding a geopoly and a json1 feature to the port.

- What does your PR fix? Fixes #
   Only adding a small new feature. Not fixing anything.

- Which triplets are supported/not supported? Have you updated the CI baseline?
   All triplets are supported.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes I believe so.

### CODE REVIEW

File: ports/sqlite3/CONTROL [hunk 1]
```diff
@@ -5,3 +5,9 @@ Description: SQLite is a software library that implements a self-contained, serv
```
Comment (LilyWangL) [inline ports/sqlite3/CONTROL:5]: Thanks for your PR! Can you please correct the Version field in the CONTROL file? You can get more information from [maintainer-guide.md](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md).

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/sqlite3/CONTROL (hunk 1 above)
Comment (sanjaychugh) [reply ports/sqlite3/CONTROL:5]: Sorry, didn't remember to update the version.
If I understood the convention properly, I have changed the version from 3.31.1-1 to 3.31.1-2

**Comment by strega-nil** [comment]
@sanjaychugh I'd like to merge this, but it needs to be merged with master first; can you do that?

**Comment by sanjaychugh** [comment]
Hi, yes I can merge master into my branch. I'll try to get that done shortly probably in the morning.

**Pull request merged**